### PR TITLE
docs(en-US): customize SIE

### PR DESCRIPTION
**Pull request by gao-sun**
add the English version of "Customize SIE" and minor content improvements.

### CODE REVIEW

File: docs/docs/tutorials/get-started/customize-sign-in-experience.md [hunk 1]
```diff
@@ -3,3 +3,49 @@ sidebar_position: 3
 ---
 
 # Customize sign-in experience
+
+In the "Get Started" tab, click the "Create" button on the right, and the browser will redirect to the Sign-in Experience tab.
+
+## Sign-in Preview
+
+:::tip Do you know...
+You can preview the result in real-time! No matter what config you update, the right-side UI will change accordingly in a moment.
```
Comment (demonzoo) [inline docs/docs/tutorials/get-started/customize-sign-in-experience.md:12]: nit: `No matter what you update on the left, the right side UI will immediately update to synchronize the change.`
Just for your reference

**Review by demonzoo — APPROVED** [review]

File: docs/docs/tutorials/get-started/customize-sign-in-experience.md (hunk 1 above)
Comment (gao-sun) [reply docs/docs/tutorials/get-started/customize-sign-in-experience.md:12]: you are English master

Comment (gao-sun) [reply docs/docs/tutorials/get-started/customize-sign-in-experience.md]: updated

**Review by IceHe — COMMENTED** [review]
Better align the heading style:

- title/heading case
- or sentence case?

File: docs/docs/tutorials/get-started/customize-sign-in-experience.md [hunk 2]
```diff
@@ -3,3 +3,49 @@ sidebar_position: 3
...
+You may notice the auto-generated *Brand color (Dark)* is slightly different compared to the original *Brand color*.
+
+Because Logto just took care of the visual difference between light and dark and computes a brand color exclusively for dark mode. If you are unsatisfied with the color, you are also free to customize it.
+:::
+
+## Terms of Use
+
+An app will likely have a Terms of Use in production, and it's mandatory to accept it before signing in or creating an account.
+
+Toggle on "Enable terms of use" and enter the URL to enforce this process. Logto will load it via an `<iframe>`.
```
Comment (IceHe) [inline docs/docs/tutorials/get-started/customize-sign-in-experience.md:37]: nit: better `Toggle on "Enable Terms of Use" and enter its content URL`?

Comment (gao-sun) [reply docs/docs/tutorials/get-started/customize-sign-in-experience.md:37]: not sure since the wording in AC is not locked yet

Comment (gao-sun) [reply docs/docs/tutorials/get-started/customize-sign-in-experience.md:37]: LOG-3069

**Pull request merged**